client/src/pages/Landing/LinkCard.tsx

```diff
@@ -0,0 +1,21 @@
+import { Link } from "react-router-dom";
+
+interface Props {
+  label: string;
+  path: string;
+  children: JSX.Element;
+}
+
+function LinkCard({ label, path, children }: Props) {
+  return (
+    <Link
+      className="flex h-1/2 min-h-[14.5rem] w-full animate-cardHoverOff flex-col items-center justify-center gap-6 rounded-xl border-2 border-sky-300 bg-sc-blue text-white hover:animate-cardHover hover:bg-sc-dark-blue md:h-full md:min-h-[27.5rem] md:gap-10"
+      to={path}
+    >
+      {children}
+      <h2 className="text-2xl uppercase md:text-3xl">{label}</h2>
```

Comment:
Do we want to use Typography (from react-mui) instead of h2? Might be worth using Typography to keep the font and size consistent throughout the project.

Reply:
I think that would also be the case if we used the default tailwind headers but this is a point worth discussing. I know Jayde and Josh are also working on UI pages as well. Let's bring this up tomorrow at the standup meeting on which way we want to go incase they did it a specific way as well.

Reply:
Good catch btw, saw your ticket but didn't notice you used MUI Typography. Better we collectively decide on a standard earlier than later!